config/samples/csi.ibm.com_v1_ibmblockcsi_cr.yaml

```diff
@@ -45,6 +45,24 @@ spec:
                     - s390x
                     - ppc64le
 
+   # callHome is a statefulSet with ibm-block-csi-call-home container
+#  callHome:
+#    repository: stg-artifactory.xiv.ibm.com:5030/ibm-block-csi-call-home-amd64
+#    tag: "1.9.0"
+#    imagePullPolicy: IfNotPresent
+#    secretName: demo-secret
```

Comment:
should we have a separate secret example+name for this feature?

Reply:
we might, anyway its for the controller and can do in the script pr scope

Reply:
which controller? what script PR?

Reply:
I ment for the driver reposetory
I thought the script will be there..
when I will open it

Reply:
it could, though it's not really part of the CSI driver.
then could you please use `demo-call-home-secret` for now, in this PR?
we can also consider renaming the existing `demo-secret` to e.g. `demo-csi-secret` (not in this PR)

Reply:
done
CSI-3929